components/ActivitiesList/ActivityCardPreview/index.js

```diff
@@ -0,0 +1,39 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+const StyledCateogriesUl = styled.ul`
+  list-style: none;
+  padding: 5px;
+  display: flex;
+  justify-content: center;
+  gap: 10px;
+`;
+
+const StyledCategory = styled.li`
+  border: 1px solid grey;
+  padding: 5px;
+  border-radius: 5px;
+  margin-bottom: 10px;
+`;
+
+export default function ActivityCardPreview({ title, categories, id, img }) {
```

Comment:
You could also pass the activity object here and destructure it in the component:

```suggestion
export default function ActivityCardPreview({ activity }) {
const { title, categories, id, img } = activity;
```

just a suggestion :)